### PR TITLE
feat(readme): add maxHeight prop to ReadmeCard component

### PR DESCRIPTION
**Pull request by tdabasinskas**
## Hey, I just made a Pull Request!

Enhance the ReadmeCard component to allow custom max height settings. This change provides more flexibility in controlling the card's appearance across different layouts and screen sizes.

### Context

The new maxHeight prop allows users to override the default height of 235px for non-fullHeight variants, enabling better customization of the component's vertical space.

### Checklist before requesting a review

- [X] I have performed a self-review of my own code
- [X] I have verified that the code builds perfectly fine on my local system
- [X] I have verified that my code follows the style already available in the repository
- [x] A changeset describing the change and affected packages. ([more info](https://github.com/AxisCommunications/backstage-plugins/blob/main/CONTRIBUTING.md#changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)

### CODE REVIEW

File: plugins/readme/src/components/ReadmeCard/ReadmeCard.tsx [hunk 1]
```diff
@@ -14,11 +14,12 @@ import { ReadmeDialog } from '../ReadmeDialog/ReadmeDialog';
 
 export type ReadmeCardProps = {
   variant?: InfoCardVariants;
+  maxHeight?: string
```
Comment (fridajac) [inline plugins/readme/src/components/ReadmeCard/ReadmeCard.tsx:17]: Can we make this type string | number? To make it similar to how Material UI works.

**Comment by fridajac** [comment]
Hi @tdabasinskas! Thank you for this PR. We will have a look at it as soon as we can :100:

File: plugins/readme/src/components/ReadmeCard/ReadmeCard.tsx [hunk 2]
```diff
@@ -14,11 +14,12 @@ import { ReadmeDialog } from '../ReadmeDialog/ReadmeDialog';
 
 export type ReadmeCardProps = {
   variant?: InfoCardVariants;
+  maxHeight?: string | number;
 };
 
 export const ReadmeCard = (props: ReadmeCardProps) => {
-  const { variant = 'gridItem' } = props;
-  const maxHeight = variant === 'fullHeight' ? 'none' : '235px';
+  const { variant = 'gridItem', maxHeight: propMaxHeight } = props;
```
Comment (fridajac) [inline plugins/readme/src/components/ReadmeCard/ReadmeCard.tsx:21]: Super small thing, but why not just call propMaxHeight maxHeight? Clear enough without the "prop"-part.

Comment (tdabasinskas) [reply plugins/readme/src/components/ReadmeCard/ReadmeCard.tsx:21]: Wouldn't it conflict with `maxHeight` we already have?

**Comment by fridajac** [comment]
Since you are updating the api of the ReadmeCard component, you also have to run the yarn build:api-reports plugins/readme command to update the api-report :)

**Comment by tdabasinskas** [comment]
> Since you are updating the api of the ReadmeCard component, you also have to run the yarn build:api-reports plugins/readme command to update the api-report :)

Done!

**Review by fridajac — APPROVED** [review]
Thank you for your contribution! Very appreciated :100:

**Comment by tdabasinskas** [comment]
No probs. Will you be creating a new release with these changes, @fridajac?

**Pull request merged**